src/main/xerus/monstercat/tabs/TabSettings.kt

```diff
@@ -80,6 +77,20 @@ class TabSettings: VTab() {
 			Settings.PLAYERSEEKBARHEIGHT.bind(valueProperty() as ObservableValue<out Double>)
 		})
 		
+		val connectionSpeed = ComboBox<String>(FXCollections.observableArrayList())
```

Comment:
I think createComboBox could be used here as well, or?

Reply:
Of course it can :)

Reply:
b2fee4677ea50675385243d7ed818a6c5a44629f

Reply:
a257af08f88e53770755f7ed4be2003b9fabd7d0